demisto_sdk/tests/integration_tests/format_test.py

```diff
@@ -0,0 +1,227 @@
+from demisto_sdk.tests.constants_test import (
+    CONF_JSON_MOCK_PATH, DESTINATION_FORMAT_INTEGRATION_COPY,
+    DESTINATION_FORMAT_PLAYBOOK_COPY, DESTINATION_FORMAT_SCRIPT_COPY, GIT_ROOT,
+    INTEGRATION_WITH_TEST_PLAYBOOKS, PLAYBOOK_WITH_TEST_PLAYBOOKS,
+    SCRIPT_WITH_TEST_PLAYBOOKS, SOURCE_FORMAT_INTEGRATION_COPY,
+    SOURCE_FORMAT_PLAYBOOK_COPY, SOURCE_FORMAT_SCRIPT_COPY)
+
+BASIC_YML_TEST_PACKS = [
+    (SOURCE_FORMAT_INTEGRATION_COPY, DESTINATION_FORMAT_INTEGRATION_COPY, IntegrationYMLFormat, 'New Integration_copy',
+     'integration'),
+    (SOURCE_FORMAT_SCRIPT_COPY, DESTINATION_FORMAT_SCRIPT_COPY, ScriptYMLFormat, 'New_script_copy', 'script'),
+    (SOURCE_FORMAT_PLAYBOOK_COPY, DESTINATION_FORMAT_PLAYBOOK_COPY, PlaybookYMLFormat, 'File Enrichment-GenericV2_copy',
+     'playbook')
+]
+
+YML_FILES_WITH_TEST_PLAYBOOKS = [
+    (
+        INTEGRATION_WITH_TEST_PLAYBOOKS,
+        DESTINATION_FORMAT_INTEGRATION_COPY,
+        IntegrationYMLFormat,
+        'New Integration',
+        'integration'),
+    (
+        SCRIPT_WITH_TEST_PLAYBOOKS,
+        DESTINATION_FORMAT_SCRIPT_COPY,
+        ScriptYMLFormat,
+        'New_script_copy',
+        'script'),
+    (
+        PLAYBOOK_WITH_TEST_PLAYBOOKS,
+        DESTINATION_FORMAT_PLAYBOOK_COPY,
+        PlaybookYMLFormat,
+        'File Enrichment-GenericV2_copy',
+        'playbook'
+    )
+]
+FORMAT_CMD = "format"
+CONF_JSON_ORIGINAL_CONTENT = {
+    "tests": [
+        {
+            "integrations": "PagerDuty v2",
+            "playbookID": "PagerDuty Test"
+        },
+        {
+            "integrations": "Account Enrichment",
+            "playbookID": "PagerDuty Test"
+        },
+        {
+            "integrations": "TestCreateDuplicates",
+            "playbookID": "PagerDuty Test"
+        }
+    ]
+}
+
+
+@pytest.mark.parametrize('source_path, destination_path, formatter, yml_title, file_type', BASIC_YML_TEST_PACKS)
+def test_formatting_yml_with_no_test_positive(source_path, destination_path, formatter, yml_title, file_type):
+    # type: (str, str, BaseUpdateYML, str) -> None
+    """
+        Given
+        - A yml file (integration, playbook or script) with no 'tests' configured
+
+        When
+        - Entering 'Y' into the prompt message about that asks the user if he wants to add 'No tests' to the file
+
+        Then
+        -  Ensure no exception is raised
+        -  Ensure 'No tests' is added in the first time
+        -  Ensure message is not prompt in the second time
+    """
+    saved_file_path = os.path.join(os.path.dirname(source_path), os.path.basename(destination_path))
+    runner = CliRunner()
+    # Running format in the first time
+    result = runner.invoke(main, [FORMAT_CMD, '-i', source_path, '-o', saved_file_path], input='Y')
+    prompt = f'The file {source_path} has no test playbooks configured. Do you want to configure it with "No tests"'
+    assert not result.exception
```

Comment:
assert not result.stderr

Reply:
I get 
```
stderr not separately captured
```

Reply:
use `runner = CliRunner(mix_stderr=False)`